modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java

```diff
@@ -2675,6 +2726,13 @@ else if (cpDefinitionOptionRel.isRequired()) {
 
 				_validateRequiredOption(cpDefinitionOptionRel, optionJSONArray);
 			}
+			else if (!cpDefinitionOptionValueRels.isEmpty()) {
+				_validateBundleInformation(
```

Comment:
These options, for whatever reason, are not listed with their CPInstance in CPInstanceOptionValueRel so we need to do this to find them unfortunately.